Utility.cpp

```diff
@@ -1,15 +1,20 @@
 #include "Utility.h"
 
+#include "Character.h"
+#include "DragonSlayer.h"
+#include "Dragon.h"
+#include "Dwarf.h"
+#include "Paladin.h"
+#include "Item.h"
 #include "HelpfulItem.h"
 #include "DefensiveItem.h"
-#include "Character.h"
-
+#include "AttackItem.h"
 
 std::vector<std::unique_ptr<Item>> makeHelpfulItems(int num)
 {
     std::vector<std::unique_ptr<Item>> items;
     
-    while( num-- >= 0 )
+    while( --num >= 0 )
```

Comment:
Why did you change this?
- [ ] revise.

Reply:
fixed number of helpful and defense items for each character.